src/Game/Game.h

```diff
@@ -110,10 +117,11 @@ class Game
     bool _quit = false;
     SDL_Event _event;
     bool _initialized = false;
+    std::unique_ptr<InputController> inputController_;
```

Comment:
It seems reasonable to support the existing code style, and this is minor issue.
So I will fix naming.
But I you will want to introduce a code convention, I recommend you to look at existing one, like Google CC or boost etc, and get the best parts =)